source/.editorconfig

```diff
@@ -0,0 +1,133 @@
+# F0.EditorConfig.DotNet
+root = true
+
+[*]
+end_of_line = crlf
+tab_width = 4
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.sln]
+charset = utf-8-bom
+indent_style = tab
+#indent_size = 4
+
+[*.{csproj,vbproj,props,targets}]
+charset = utf-8
+indent_style = space
+indent_size = 2
+
+[*.{cs,csx,vb,vbx}]
+charset = utf-8-bom
+indent_style = tab
+#indent_size = 4
+max_line_length = 130
+
+[*.xaml]
+charset = utf-8-bom
+indent_style = tab
+#indent_size = 4
+
+[*.xml]
+indent_style = space
```

Comment:
Intentional that we have set space here although we use tabs in everything else? Feel like we should be consistent here

Reply:
Changed to _tabs over spaces_ for every file.